### DISCUSSION TITLE
What is the meaning of Err: Accuracy (-44.961) is lower than Zero, which means that the RMSE (in latest 24 hours) of the 0t

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#184] [BaiduKDDCup2022] [answered]
Hello,

What is accuracy and why is negative?
Err: Accuracy (-44.961) is lower than Zero, which means that the RMSE (in latest 24 hours) of the 0t

Thank you

**Comment by fersebas** [comment]
Could I see the full log? Team: datateam-UCM. Thanks

**Comment by ramber1836** [comment, marked as answer]
The full log goes like this: 


The experimental settings are: 
{'checkpoints': 'checkpoints', 'start_col': 3, 'pred_file': 'predict.py', 'framework': 'base'}

	The 0-th prediction -- RMSE: 3665.910607247619, MAE: 4965.754114735479, and Accuracy: -44.96071078038493
######{"code": -1, "errorMsg": "Err: Accuracy (-44.961) is lower than Zero, which means that the RMSE (in latest 24 hours) of the 0th prediction is too large!! (Archive.zip)", "data": [{"submitFile": "Archive.zip"}, {"platform": "base"}]}
Traceback (most recent call last):
  File "/home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/base/evaluation.py", line 332, in eval
    return evaluate(tmp_dir)
  File "/home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/base/evaluation.py", line 252, in evaluate
    "is too large!".format(tmp_acc, i))
evaluation.EvaluationError: Accuracy (-44.961) is lower than Zero, which means that the RMSE (in latest 24 hours) of the 0th prediction is too large!

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/aistudio/eval_main.py", line 83, in main
    ret = evaluation.eval(submit_file)
  File "/home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/base/evaluation.py", line 336, in eval
    raise Exception(msg)
Exception: Err: Accuracy (-44.961) is lower than Zero, which means that the RMSE (in latest 24 hours) of the 0th prediction is too large!! (Archive.zip)

**Reply by fersebas** [reply]
Thank you, make sense
